config/locales/client.en.yml

```diff
@@ -873,11 +873,17 @@ en:
           prefill:
             title: "Prefill with settings for:"
             gmail: "Gmail"
+            outlook: "Outlook"
+            office365: "Office365"
```

Comment:
Microsoft renamed that to "Microsoft 365", so maybe use the current name in the UI?

Reply:
Yup will change 👍